### DISCUSSION TITLE
Save the speed settings in the URL?

### DISCUSSION THREAD

**Discussion by digitalurban** [benfl3713/DepartureBoard#275] [Q&A]
A brilliant site! its almost perfect - i was looking to load the site on a pi and move through a few other sites automatically (ie a dashboard) but it looks like i need to set up the speed settings each time i boot.

Is there a way to embed the speed settings into the url, so i can simply load the site and the scroll will be at the normal speed (50) - the default seems very fast.

Andy

**Comment by benfl3713** [comment]
Hi @digitalurban nice to hear from you and that you have found this project. What you're requesting does not exist right now but is very simple to add, I'm sure I can add this in for you today and let you know when it's live

**Comment by benfl3713** [comment]
I've added this in now, you should be able to use the following url parameter: `?scrollSpeed=50`

**Reply by digitalurban** [reply]
Amazing! thank you ...

Andy
